README.md

```diff
@@ -0,0 +1,20 @@
+An approach for integrating PennyLane with Orquestra.
```

Comment:
Perhaps we can find a better word than "approach"?

"plugin"? "package"?

Reply:
Fair point, for sure! Update to `package`. (The `README` will also be overhauled when adding the docs.)